tests/test_filtering.py

```diff
@@ -62,6 +64,13 @@ class WidgetListView(GenericModelView):
 
         filtering = Filtering(
             color=operator.eq,
```

Comment:
One way would be for Filtering to inject it after initialization. You shouldn't share a filter field instance in the same way you shouldn't share a sql alchemy column, the object can only be bound once